config/initializers/holidays.rb

```diff
@@ -0,0 +1,7 @@
+CAB_HOLIDAYS = [
```

Comment:
`CAB_HOLIDAYS = Array(Date.parse('2016/12/23')..Date.parse('2017/01/02'))` is more concise.